freqtrade/rpc/rpc.py

```diff
@@ -225,22 +228,23 @@ def _rpc_trade_statistics(
 
         # FIX: we want to keep fiatconverter in a state/environment,
```

Comment:
Probably should remove this comment also?

Reply:
good catch